Readme.md

```diff
@@ -80,7 +80,7 @@ app.use((ctx, next) => {
 
 ### GeneratorFunction
 
-To use generator functions, you must use a wrapper such as [co](https://github.com/tj/co) that is no longer supplied with Koa.
+To use generator functions, you must use a wrapper such as [co](https://github.com/tj/co).
```

Comment:
After thinking about it, I agree.  The overview document should focus on the recommended way, which I think now is async and leave the other two formats for deeper documentation.  I'd like to submit that as a separate PR, though, so as to not block the merging of this migration guide.